### PR TITLE
Reformulated the exercise 2 on Quiz 12

### PR DESCRIPTION
**Pull request by y-martinez**
## Summary

Following the @NicolasSandoval suggestions, I reformulated the statement of exercise 2, now it is more understandable.

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

### CODE REVIEW

**Review by angelasofiaremolinagutierrez — APPROVED** [review]
LGTM! :)

**Review by LeoCumpli21 — APPROVED** [review]

File: _sources/quiz/Quiz12.rst [hunk 1]
```diff
@@ -67,30 +68,36 @@ Ejercicio 2
...
+                "El televisor es bueno",
+                "Esperado: El televisor es bueno"
             )
             self.assertEqual(
-                no_es_malo("Esta cena no es malo!"),
-                "Esta cena bueno!",
-                "Esperado: Esta cena bueno!"
+                no_es_malo("El asado de la cena no es malo!"),
+                "El asado de la cena es bueno!",
+                "Esperado: El asado de la cena es bueno!!"
```
Comment (eamanu) [inline _sources/quiz/Quiz12.rst:78]: The last `!` is wrong, isn't?

Comment (y-martinez) [reply _sources/quiz/Quiz12.rst:78]: Yeah! good catch 👍🏼

Comment (y-martinez) [reply _sources/quiz/Quiz12.rst:78]:
```suggestion
                "Esperado: El asado de la cena es bueno!"
```

**Review by NicolasSandoval — APPROVED** [review]
Great! Thanks.

**Pull request merged**